Argument of type 'IconPack' is not assignable to parameter of type 'IconDefinitionOrPack'

### Describe the problem

Trying to import fab icons to angular project, but cli says there's the following error:

`error TS2345: Argument of type 'IconPack' is not assignable to parameter of type 'IconDefinitionOrPack'. Type 'IconPack' is missing the following properties from type 'IconDefinition': icon, prefix, iconName
`

--- a/.github/workflows/check.yml
+++ b/.github/workflows/check.yml
@@ -14,6 +14,7 @@ jobs:
           path: .yarn/cache
           key: ${{ hashFiles('yarn.lock') }}
       - run: yarn
+      - run: yarn add -D @fortawesome/fontawesome-svg-core@1.2.36 @fortawesome/free-regular-svg-icons@5.15.4 @fortawesome/free-solid-svg-icons@5.15.4
       - run: yarn format:enforce
       - run: yarn lint
       - run: yarn test
@@ -36,7 +37,6 @@ jobs:
           path: .yarn/cache
           key: ${{ hashFiles('yarn.lock') }}
       - run: yarn
-      - run: yarn add -D @fortawesome/fontawesome-svg-core@1.3.0-beta2 @fortawesome/free-regular-svg-icons@6.0.0-beta2 @fortawesome/free-solid-svg-icons@6.0.0-beta2
       - run: yarn format:enforce
       - run: yarn lint
       - run: yarn test

--- a/package.json
+++ b/package.json
@@ -41,9 +41,9 @@
     "@angular/language-service": "~13.3.0",
     "@angular/platform-browser": "~13.3.0",
     "@angular/platform-browser-dynamic": "~13.3.0",
-    "@fortawesome/fontawesome-svg-core": "~1.2.36",
-    "@fortawesome/free-regular-svg-icons": "^5.15.4",
-    "@fortawesome/free-solid-svg-icons": "^5.15.4",
+    "@fortawesome/fontawesome-svg-core": "^6.1.0",
+    "@fortawesome/free-regular-svg-icons": "^6.1.0",
+    "@fortawesome/free-solid-svg-icons": "^6.1.0",
     "@types/jasmine": "~4.0.0",
     "@types/node": "~16.11.26",
     "@typescript-eslint/eslint-plugin": "5.15.0",
@@ -80,7 +80,7 @@
     "svg"
   ],
   "peerDependencies": {
-    "@fortawesome/fontawesome-svg-core": "~1.2.27 || ~1.3.0-beta2"
+    "@fortawesome/fontawesome-svg-core": "~1.2.27 || ~1.3.0-beta2 || ^6.1.0"
   },
   "schematics": "./schematics/collection.json",
   "packageManager": "yarn@3.2.0"

